Azure/communication-ui-library-android
# [Features] alternate theme feature

## Purpose

* Allows selection of default or alternate theme when launching the call Composite

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code

1) Select feature from checklist (Secondary theme)
2) Make sure the primary color is different (should be teal).

## What to Check
Verify that the following are valid
* ...

## Other Information
